client/nsxt_client.go

```diff
@@ -64,6 +64,11 @@ func (c *nsxtClient) GetTransportNodeStatus(nodeID string) (manager.TransportNod
 	return transportNodeStatus, err
 }
 
+func (c *nsxtClient) ListEdgeClusters() (manager.EdgeClusterListResult, error) {
+	edgeClustersResult, _, err := c.apiClient.NetworkTransportApi.ListEdgeClusters(c.apiClient.Context, nil)
```

Comment:
ListEdgeClusters result could be paginated, need to pass cursor parameter

Reply:
fixed in 0d26e32